api/db/seed/wage_chunk.json

```diff
@@ -0,0 +1,65 @@
+[
+    {
+        "hours": 54,
+        "rate": 42.25,
+        "start_date": "2018-9-10",
+        "end_date": "2018-12-10",
+        "assignment_id": 0
```

Comment:
The ids shouldn't be hardcoded...they should be read from the return value of the api call and synthesized on the spot. Eventually the controller will generate randomized ids, and then this seeing will fail.

Reply:
Actually, this only looks like `id`s it's actually the index of entry from `assignment.json`.

Reply:
Oh. That's pretty confusing then...If it's separate linking data, put it in a separate JSON object. That way the seed data looks like seed data.

Reply:
Do you mean like in a separate file? I'm having trouble imagining how that would work.

Reply:
Yes. Maybe `wage_chunk_links.json`. You're essentially creating a DSL for batch-processing with the API right now...

Reply:
Is the DSL comparison what I'm doing right now or what I should/should not be attempting? Also, the seeding should be done via chaining API in order(?)

Reply:
@michellemtchai Exactly! You're chaining the API calls. If you want to create a mini DSL to assist you with chaining a bunch of calls in a particular order, that's fine. If you want to hard code it, that's also fine.

Reply:
Okay. I think I get it now. Thanks.